saleor/graphql/warehouse/mutations.py

```diff
@@ -48,12 +51,8 @@ def clean_input(cls, info, instance, data, input_cls=None):
                 error.code = WarehouseErrorCode.REQUIRED.value
                 raise ValidationError({"name": error})
 
-        shipping_zones = cleaned_input.get("shipping_zones", [])
-        if not validate_warehouse_count(shipping_zones, instance):
-            msg = "Shipping zone can be assigned only to one warehouse."
-            raise ValidationError(
-                {"shipping_zones": msg}, code=WarehouseErrorCode.INVALID
-            )
+        # assigning shipping zones in the WarehouseCreate mutation is deprecated
+        cleaned_input.pop("shipping_zones", None)
```

Comment:
Isn't it a breaking change? We leave an input but we change the behaviour of using it.

Reply:
I do not think so, it's a fix for the feature that was introduced in 3.5. And it shouldn't be possible of doing such a thing. It should be blocked in 3.5.

Reply:
We should mention this as a breaking change in the changelog, this behavior was here for since many versions back.

Reply:
Can we at least have it reject the input and not ignore it silently?

Reply:
Done.